### ISSUE TITLE
[docs] 

### ISSUE THREAD

**Issue by thgh** [expo/expo#23327]
### Summary

Link does not work, it should be .mdx instead of .md

https://github.com/expo/fyi/blob/main/expo-extension-migration.md

https://github.com/expo/fyi/blob/main/expo-extension-migration.mdx

### Link to the related docs page

https://docs.expo.dev/bare/using-expo-client/#deprecated-expo-extensions

**Comment by RRaideRR** [comment]
Good catch @thgh. I'm not belonging to the expo team but do you know that you can edit that page by your own? Via the "Edit this page" button at the bottom. Then you can create a PR - its fairly easy and the expo team tries to merge these things quite fast :-).

**Comment by amandeepmittal** [comment]
Good catch @thgh! We've initiated a PR to fix it. Thank you!

@RRaideRR 🫶
